ts/Grid/Core/Credits.ts

```diff
@@ -79,9 +95,8 @@ class Credits {
      * @param grid
      * The Grid instance which the credits belong to.
```

Comment:
missing `options` param

Reply:
It's intentional we do not need options here.